Utilities/import-llvm

```diff
@@ -15,41 +15,55 @@ import sys
 import subprocess
 
 # ADT types.
-ADT_imports = ['EpochTracker', 'iterator', 'iterator_range', 'Hashing', 'None', 'Optional',
-               'Statistic', 'StringExtras', 'STLExtras', 'AllocatorList', 'Triple']
+ADT_imports = ['iterator', 'iterator_range', 'APFloat', 'APInt', 'APSInt',
+               'AllocatorList', 'EpochTracker', 'Hashing', 'None', 'Optional',
+               'Statistic', 'StringExtras', 'STLExtras']
```

Comment:
These lists are getting quite messy.  I think we're better off just making each variable into a single list, and putting every dependency on its own line.  That will make it much easier to see changes in the future.  Splitting it up "thematically" like this made more sense when we had fewer imports and you could still conceivable eyeball changes.